python/pyspark/sql/functions.py

```diff
@@ -1191,8 +1192,7 @@ def months_between(date1, date2, roundOff=True):
 @since(2.2)
 def to_date(col, format=None):
     """Converts a :class:`Column` into :class:`pyspark.sql.types.DateType`
-    using the optionally specified format. Specify formats according to
-    `DateTimeFormatter <https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html>`_. # noqa
+    using the optionally specified format. Specify formats according to `datetime pattern`_. # noqa
```

Comment:
does `datetime pattern`_ work here? The URL is not specified in this method.

Reply:
I checked the `partition discovery` here http://spark.apache.org/docs/3.0.0-preview2/api/python/pyspark.sql.html#module-pyspark.sql.streaming. Some take effect but others don't. BUT ALSO /xxx.html seems not to point to the right page.

Reply:
The full url is rewritten to http://spark.apache.org/sql-data-sources-parquet.html#partition-discovery